src/core/Pricing/data.tsx

```diff
@@ -14,7 +14,7 @@ export const planData: PricingDataFeature[] = [
       color: "text-neutral-500",
     },
     price: { amount: 0 },
-    cta: { text: "Start for free", url: "#" },
+    cta: { text: "Start for free", url: "https://ably.com/sign-up" },
```

Comment:
Could we drop `https://ably.com` from these links, unless it is for the stories only.

Reply:
Can change this, I'm not worried about link health on stories